tests/foreman/cli/test_repository.py

```diff
@@ -228,8 +229,14 @@ def test_positive_create_with_default_download_policy(self):
 
         @Assert: YUM repository with a default download policy
         """
+        default_dl_policy = entities.Setting().search(
```

Comment:
I don't care, will fix it your way then. Just an argument on
> Quick example why - when some API path is broken i'm expecting to see corresponding API tests failing and i'm definitely not ok with CLI tests failing because of that.

hammer calls API in the background